specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -3788,6 +3973,156 @@
+      ],
+      "description": "The storage account blob inventory policy properties."
+    },
+    "BlobInventoryPolicySchema": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Policy is enabled if set to true."
+        },
+        "destination": {
+          "type": "string",
+          "description": "Container name where blob inventory files are stored. Must be pre-created."
+        },
+        "type": {
+          "type": "string",
+          "description": "The valid value is Inventory",
+          "enum": [
+            "Inventory"
+          ],
+          "x-ms-enum": {
+            "name": "InventoryRuleType",
+            "modelAsString": true
+          }
+        },
+        "rules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BlobInventoryPolicyRule"
+          },
+          "description": "The storage account blob inventory policy rules. The rule is applied when it is enabled."
+        }
+      },
+      "required": [
+        "enabled",
+        "destination",
+        "type",
+        "rules"
+      ],
+      "description": "The storage account blob inventory policy rules."
+    },
+    "BlobInventoryPolicyRule": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Rule is enabled when set to true."
+        },
+        "name": {
+          "type": "string",
+          "description": "A rule name can contain any combination of alpha numeric characters. Rule name is case-sensitive. It must be unique within a policy."
+        },
+        "definition": {
+          "$ref": "#/definitions/BlobInventoryPolicyDefinition",
+          "description": "An object that defines the blob inventory policy rule."
+        }
+      },
+      "required": [
+        "name",
+        "enabled",
+        "definition"
+      ],
+      "description": "An object that wraps the blob inventory rule. Each rule is uniquely defined by name."
+    },
+    "BlobInventoryPolicyDefinition": {
+      "properties": {
+        "filters": {
+          "$ref": "#/definitions/BlobInventoryPolicyFilter",
+          "description": "An object that defines the filter set."
+        }
+      },
+      "required": [
+        "filters"
+      ],
+      "description": "An object that defines the blob inventory rule. Each definition consists of a set of filters."
+    },
+    "BlobInventoryPolicyFilter": {
+      "properties": {
+        "prefixMatch": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "An array of strings for blob prefixes to be matched."
+        },
+        "blobTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "An array of predefined enum values. Valid values include blockBlob, appendBlob, pageBlob. Hns accounts does not support pageBlobs."
```

Comment:
can we limit user only input the correct value?

Reply:
Wei Server-side has the validation, we are following the same approach we used for Management policy LCM

Reply:
To avoid SDK user from input wrong value, it would bettter to limit user only input the correct value.

Reply:
For now wil be sticking to this approach which we have also followed for LCM. Not very clear how  to limit SDK user with correct values. Server side has the validations in place